README/ReleaseNotes/v610/index.md

```diff
@@ -248,6 +248,7 @@ The following interfaces have been removed, after deprecation in v6.08.
 ## Parallelism and PROOF
 - Add ROOT::GetImplicitMTPoolSize function to get the size of the pool used to enable implicit multi threading
 - Add the TThreadExecutor::Foreach method for parallelising functions featuring void return type
+- Add TBufferMerger and TBufferMergerFile to facilitate writing in parallel to a single output file
```

Comment:
@dpiparo Could you please let us know what are your thoughts here?

@pcanal I only see two paragraphs other than the header line. Do you mean to omit the origin of the classes? If yes, why?